great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -374,36 +383,474 @@ def apply_theme(
         theme = DataAssistantResult._get_theme(theme=theme)
         return [chart.configure(**theme) for chart in charts]
 
+    @staticmethod
+    def get_nominal_metric_chart(
+        df: pd.DataFrame,
+        metric_name: str,
+        sequential: bool,
+        subtitle: Optional[str] = None,
+    ) -> alt.Chart:
+        """
+        Args:
+            df: A pandas dataframe containing the data to be plotted
+            metric_name: The name of the metric as it exists in the pandas dataframe
+            sequential: Whether batches are sequential in nature
+            subtitle: The subtitle, if applicable
+
+        Returns:
+            An altair chart for nominal metrics
+        """
+        batch_name: str = "batch"
+        batch_identifiers: List[str] = [
+            column for column in df.columns if column not in [metric_name, batch_name]
+        ]
+        batch_type: alt.StandardType
+        if sequential:
+            batch_type = AltairDataTypes.ORDINAL.value
+        else:
+            batch_type = AltairDataTypes.NOMINAL.value
+        batch_component: BatchPlotComponent = BatchPlotComponent(
+            name=batch_name,
+            alt_type=batch_type,
+            batch_identifiers=batch_identifiers,
+        )
+
+        column_number: str = "column_number"
+
+        metric_type: alt.StandardType = AltairDataTypes.NOMINAL.value
+        column_set: List[str]
+        metric_component: MetricPlotComponent
```

Comment:
Could we rename this to `metric_plot_component` to be consistent? I just wouldn't want anyone to be confused that it is a component of the metric itself.

Reply:
These renames apply in a few other places as well, let me know if it helps if I add more comments.

Reply:
I will take care of this in the follow-up PR: https://github.com/great-expectations/great_expectations/pull/5259